guides/rails_guides/generator.rb

```diff
@@ -105,6 +106,10 @@ def select_only(guides)
         end
       end
 
+      def process_scss
+        system "sass ./assets/stylesrc/style.scss:./assets/stylesheets/style.css ./assets/stylesrc/highlight.scss:./assets/stylesheets/highlight.css"
```

Comment:
I don't think we have sass in the machine that generates the guides and I'd like to avoid installing another dependency. Do we need to run the sass given the css is also present in the repository?

Reply:
Ah, carlos's PR solve this problem.